docs/INSTALLATION_LOCAL.md

```diff
@@ -1,11 +1,15 @@
 # How to install the Open Event Web App Generator on my local machine

```

Comment:
@dexta11 I don't know what does `my machine` refers to user. It should be OS specific.
`How to install the Open Event Web App Generator on my local machine`